packages/devtools/lib/src/memory/memory.dart

```diff
@@ -234,25 +476,63 @@ class MemoryScreen extends Screen with SetStateMixin {
     }
   }
 
+  List<String> _knownSnapshotClasses = [];
+  List<String> getAllClassesKnown() {
+    if (_knownSnapshotClasses.isEmpty) {
+      final List<ClassHeapDetailStats> classesData = tableStack.first.data;
+      for (ClassHeapDetailStats stat in classesData) {
+        _knownSnapshotClasses.add(stat.classRef.name);
+      }
+    }
+
+    return _knownSnapshotClasses;
+  }
+
+  Future<void> _search() async {
+    ga.select(ga.memory, ga.search);
+
+    // Subsequent snapshots will reset heapPopupList to empty.
+    if (heapPopupList.isEmpty) {
+      // Only fetch once between snapshots.
+      heapPopupList.setList(getAllClassesKnown());
+    }
+
+    if (vmSearchField.element.style.visibility != 'visible') {
```

Comment:
seems like `isVisible` could be a nice helper to add to the CoreElements class

Reply:
Good idea - added

  bool get isVisible => element.style.visibility == 'visible';